client/tx/factory.go

```diff
@@ -471,14 +471,19 @@ func (f Factory) Prepare(clientCtx client.Context) (Factory, error) {
 	}
```

Comment:
From an offline standpoint, I think, we should not even try the above right?
So, if in offline mode, we simply return the factory early? and keep the `||`?

Reply:
SGTM

Reply:
Alright, doing that then.